release.config.js

```diff
@@ -63,5 +63,5 @@ module.exports = {
 };
 
 function equalsIgnoreCase(a, b) {
-  return a.localeCompare(b, 'en', { sensitivity: 'base' })
-}
\ No newline at end of file
+  return a.localeCompare(b, 'en', { sensitivity: 'base' }) == 0
```

Comment:
I get warnings to use `===` instead of `==` since it's more strict, but I don't think it matters in this case.

Reply:
Thanks @mhollisterhg , I can do that!